schema/article.schema.json

```diff
@@ -115,7 +115,7 @@
                   "type": "string",
                   "title": "Repeat",
                   "description": "Repeat-x: The background image is repeated only horizontally. Repeat-y: The background image is repeated only vertically",
-                  "default": "",
+                  "default": "no-repeat",
                   "enum": [
                     "",
```

Comment:
We should probably remove the empty string `""` from the list of options for article / block / content object schemas if we're setting a default

Reply:
Agreed and removed